### ISSUE TITLE
Wildcards not replaced inside notification "data" field

### ISSUE THREAD

**Issue by f18m** [nielsfaber/alarmo#915]
### Alarmo Version

v1.9.15

### HA Version

2024.3.3

### Bug description

When creating a notification in Alarmo, I noticed that the following YAML code does not produce the intended message:

```
service: notify.myemail
data:
  message: Alarm has been triggered
  title: Alarm has been triggered
  target: myemail@gmail.com
  data:
    html: |
      <h1>Alarm was triggered!!!</h1>
      <p>Caused by {{open_sensors}}</p>
```

The "notify.myemail" service is a simple SMTP notifier that I setup using the standard home assistant SMTP component (https://www.home-assistant.io/integrations/smtp/).
Indeed what happens is that the email I receive contains the literal "{{open_sensors}}" un-expanded in the email body.
The reason seems to be that the wildcards are expanded inside the "message" and "title" but not in the "data" field.

Can you confirm that?

### Steps to reproduce

1. setup SMTP integration
2. create a notification in Alarmo with a custom "data.html" body including an Alarmo wildcard
3. hit the "try" button in Alarmo to trigger the notification or, just trigger the condition that will trigger the notification to be sent (in my case: when the alarm fires)

**Comment by f18m** [comment]
adding comment to avoid closure

**Comment by nielsfaber** [comment]
This should be fixed in the v1.10.1 release.
Please update your Alarmo installation and try it out.
You can close the issue if it works OK for you now.
